1_fetch/src/fetch_inst_stage.R

```diff
@@ -0,0 +1,77 @@
+#' @title Download instantaneous stage if no daily stage available.
+#'
+#' @param ind_file character file name where the output should be saved
+#' @param dv_data_ind indicator file for the data.frame of dv_data
+#' @param stage_data_ind indicator file for the data.frame of sites with flood stage
+#' @param request_limit number indicating how many sites to include per dataRetrieval request (from viz_config.yml)
+fetch_inst_stage <- function(ind_file, dv_data_ind, stage_data_ind, request_limit){
+
+  dv_data <- readRDS(scipiper::sc_retrieve(dv_data_ind, remake_file = '1_fetch.yml'))
+  sites_stage <- readRDS(scipiper::sc_retrieve(stage_data_ind, remake_file = '1_fetch.yml'))
+
+  dv_data <- readRDS("1_fetch/out/dv_data.rds")
+  sites_stage <- readRDS("1_fetch/out/sites_stage.rds")
```

Comment:
it looks like maybe lines 12 and 13 were shortcuts during development but should be removed in favor of lines 9 and 10 now

Reply:
oh oops!